packages/extension-api/src/extension-api.d.ts

```diff
@@ -2493,6 +2493,22 @@ declare module '@podman-desktop/api' {
     path: string;
   }
 
+  /**
+   * Options to update CliTool instance
+   */
+  export interface CliToolUpdateOptions {
+    version: string;
+    displayName?: string;
+    markdownDescription?: string;
+    images?: ProviderImages;
+    path?: string;
```

Comment:
Should we also include the asset name, as when retrieving the / installing the new update, you'll have to pass in the asset name determined by the extension (example, hey, install arm64 darwin). 

Or should we just do quickpick / prompt for now?

Reply:
it's the extension that is doing the update so extension is aware on asset name to pick-up